crates/bevy_sprite/src/mesh2d/mesh2d.wgsl

```diff
@@ -5,31 +5,23 @@
 #import bevy_sprite::mesh2d_functions
 
 struct Vertex {
-    [[location(0)]] position: vec3<f32>;
-    [[location(1)]] normal: vec3<f32>;
-    [[location(2)]] uv: vec2<f32>;
+    @location(0) position: vec3<f32>,
+    @location(1) normal: vec3<f32>,
+    @location(2) uv: vec2<f32>,
 #ifdef VERTEX_TANGENTS
-    [[location(3)]] tangent: vec4<f32>;
+    @location(3) tangent: vec4<f32>,
 #endif
 #ifdef VERTEX_COLORS
-    [[location(4)]] colors: vec4<f32>;
+    @location(4) colors: vec4<f32>,
 #endif
```

Comment:
These are just the import.

Reply:
Technically no, those are the input from the vertex stage, they could evolve independently

Reply:
The input to the fragment stage has to match the output from the vertex stage for the parts that are interpolated (the ones labelled with a location) so I don’t understand your argument. They have to be the same.

Reply:
the one this comment is on is the input of the *vertex* stage, they could be different from the output of the vertex stage (and the input of the fragment stage)

Reply:
Ah, I lost that context from the lines I highlighted when adding my comment. Well, fair enough then.